### ISSUE TITLE
Too Large Scaled on Android

### ISSUE THREAD

**Issue by Megamichi123** [minetest/minetest#13671]
##### Minetest version

```
5.7.0
```

Active renderer:
Irrlicht device:?

##### OS / Hardware

Operating system:Android Huawei P20 Pro
CPU:?

GPU model:ß
OpenGL version:?

##### Summary

When opening the app, it suddenly looked like the picture in front of it, but it was normal and everything worked.
I've already adjusted the .conf but that doesn't help with system info but it resolves the same and minetest automatically inserts it

**Comment by Megamichi123** [comment]
![WIN_20230718_10_28_37_Pro](https://github.com/minetest/minetest/assets/125511169/19985792-3711-46cf-8912-04c7d594d34a)

**Comment by rollerozxa** [comment]
Does your phone change its resolution on the fly?

**Comment by Megamichi123** [comment]
No, automatic turning is no longer synonymousNo, automatic turning is no longer synonymous

**Comment by Zughy** [comment]
@srifqi

**Comment by grorp** [comment]
Possibly related issue: #12007

**Comment by Unacceptium** [comment]
@Megamichi123 i dont know if you and the device still present, but if yes, can you try using an app wich can force rotation? for example, "Menu button" and set a floating button for force rotation. start the game with set forced rotation to portrait mode! and try to start the game when force rotation set to landscape before opening the game! 

i comment this because i have saw very similar screen when i done that. 

a possible way to fix: use a DPI or Screen Resolution changer app, wich requires root. try setting one setup wich is similar to default. the app should force the setting to dont change on the fly, meaning the game get the same setup as the device. 

i comment this one because this exact thing happened long years before on a Lenovo A1000 i haved. possibly theres a setting for it in Engineer mode, but i dont recommend fricking these settings if you dont have pc and knowledge of debugging these settings.

**Comment by Zughy** [comment]
@Megamichi123 any updates? Have you tried Unacceptium suggestions?

**Comment by grorp** [comment]
marking this issue as "possible close" since the Android version is now based on SDL, which hopefully doesn't have such problems
